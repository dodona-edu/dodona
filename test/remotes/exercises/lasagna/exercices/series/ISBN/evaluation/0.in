5
1
7
2
0
5
3
6
9
0
4
9
3
4
0
1
4
5
7
8
8
1
8
6
3
3
8
7
6
1
5
5
8
3
7
7
2
8
7
1
1
5
4
3
8
0
1
1
2
5
3
0
9
1
9
1
9
1
9
7
9
8
6
3
5
7
7
4
3
0
7
2
3
9
1
0
4
3
7
3
9
4
9
5
5
2
5
3
8
7
5
2
0
1
9
0
2
8
9
3
5
2
2
2
7
5
8
9
2
6
7
3
7
0
6
2
5
2
8
9
8
1
4
0
0
7
4
4
4
5
4
7
7
3
9
3
7
0
0
4
5
0
6
8
0
6
7
9
8
9
8
6
5
8
2
2
9
1
5
1
3
6
1
4
9
1
9
4
0
1
3
2
1
2
2
1
7
1
0
9
4
4
6
0
4
0
2
9
4
5
0
6
7
9
2
9
2
5
2
5
6
2
9
6
5
8
2
3
9
5
7
8
3
2
3
2
0
6
6
8
7
6
2
4
7
5
6
1
3
2
5
1
7
8
0
0
6
6
9
1
9
9
3
4
3
8
9
4
1
2
5
7
5
3
6
5
8
6
2
6
6
2
3
2
0
5
0
8
9
6
8
0
0
7
6
7
9
7
0
7
2
5
1
8
0
4
5
1
8
2
6
7
3
2
5
2
3
6
4
3
5
4
7
4
3
7
1
7
0
5
7
2
0
0
0
7
8
2
7
7
3
3
6
3
6
3
6
4
3
4
9
2
2
2
2
9
1
4
4
6
8
3
1
7
0
5
3
0
8
1
6
1
4
7
7
8
1
4
3
5
4
2
3
4
8
2
8
2
5
5
8
3
2
7
5
8
0
1
6
2
1
6
5
2
8
2
6
3
3
0
9
5
9
3
8
1
2
2
2
6
8
6
7
5
6
0
7
4
3
9
1
7
6
0
9
5
3
0
9
2
6
4
1
5
6
8
6
2
0
2
1
5
5
9
6
2
2
4
9
9
6
7
2
7
5
7
7
0
8
7
