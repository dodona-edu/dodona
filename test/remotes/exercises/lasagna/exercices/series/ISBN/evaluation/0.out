7
8
2
2
4
5
2
9
7
3
8
6
9
4
7
0
4
9
1
10
7
1
5
2
0
9
2
4
10
1
7
4
9
0
5
8
6
5
3
3
4
3
2
9
2
10
3
10
8
3
----------------------------------------
python input without prompt: true
block count: multi
input block size: 9
output block size: 1 
show line numbers: false
